### PR TITLE
Fix bug 1152191: Properly serve error json for non-existent docs in children api

### PR DESCRIPTION
**Pull request by Elchi3**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1152191#c1

### CODE REVIEW

File: kuma/wiki/tests/test_views.py [hunk 1]
```diff
@@ -301,6 +301,14 @@ def _depth_test(depth, aught):
         json_obj = json.loads(resp.content)
         eq_(json_obj['subpages'][0]['title'], 'A Child')
 
+        # Test if we are serving an error json if document does not exist
+        no_doc_url = reverse('wiki.get_children', args=['nonexistentDocument'],
+            locale=settings.WIKI_DEFAULT_LANGUAGE)
+        resp = self.client.get(no_doc_url)
+        result = json.loads(resp.content)
+        aught = json.loads("""{"error": "Document does not exist."}""")
```
Comment (jezdez) [inline kuma/wiki/tests/test_views.py]: No need to json.loads the thing that you expect, just use `{'error': 'Document does not exist.'}`.

Also, what's an aught? I know [oods](http://en.wikipedia.org/wiki/Ood), are they related?

**Comment by jezdez** [comment]
LGTM other than the minor stylistic change mentioned.

File: kuma/wiki/tests/test_views.py (hunk 1 above)
Comment (Elchi3) [inline kuma/wiki/tests/test_views.py]: Aughts ought to be anything or nothing at all, but we don't know if peaceful, so I removed them from this code to protect the universe.

**Comment by jezdez** [comment]
Thanks @Elchi3!

**Pull request merged**